test/new-e2e/pkg/utils/e2e/client/host.go

```diff
@@ -126,10 +126,28 @@ func (h *Host) Execute(command string, options ...ExecuteOption) (string, error)
 	return h.executeAndReconnectOnError(command)
 }
 
-func (h *Host) executeAndReconnectOnError(command string) (string, error) {
+func (h *Host) executeAndReconnectOnError(command string) (stdout string, err error) {
 	scrubbedCommand := h.scrubber.ScrubLine(command) // scrub the command in case it contains secrets
+	defer func() {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You need to call `scrubber` also at line 159

```go
fmt.Errorf("%v: %w", stdout, err)
```

Reply:
Ah, the defer would catch it and scrub this error message